recipe/meta.yaml

```
@@ -86,66 +49,34 @@ outputs:
       - etc/conda/deactivate.d/~cuda-nvcc_deactivate.sh  # [linux]
     requirements:
       run:
-        - {{ pin_subpackage("cuda-nvcc-impl_" + target_platform, exact=True) }}
+        - cuda-nvcc-dev_{{ cross_target_platform }} {{ version }}.*
+        - cuda-cudart-dev_{{ cross_target_platform }} {{ cuda_version }}.*
+        - cuda-driver-dev_{{ cross_target_platform }} {{ cuda_version }}.*
+        - cuda-nvcc-tools {{ version }}.*
+        - cuda-nvcc-impl {{ version }}.*   # [target_platform == cross_target_platform]
+        - sysroot_{{ cross_target_platform }}  # [linux]
```

`sysroot_*` is currently used here

---

But where is the version number used?

---

```suggestion
        - pin_compatible("sysroot_" + cross_target_platform)  # [linux]
```

---

No, that wouldn't work. (It might give you 2.28, depending on what was in host). You should go back to how it was before.

---

The version in `host` would be set by the version in `conda_build_config.yaml` (pulled into the variant files)

Unfortunately what was here before was insufficiently constrained. So we are exploring how best to fix that

---

Think the original logic was too indirect for `conda-build` to follow. Would suggest we spell these out

```suggestion
        - {{ pin_compatible("sysroot_linux-64") }}       # [cross_target_platform == "linux-64"]
        - {{ pin_compatible("sysroot_linux-aarch64") }}  # [cross_target_platform == "linux-aarch64"]
        - {{ pin_compatible("sysroot_linux-ppc64le") }}  # [cross_target_platform == "linux-ppc64le"]
```